src/app/component/ProductReviewForm/ProductReviewForm.style.scss

```diff
@@ -0,0 +1,168 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+.ProductReviewForm {
+    &-Title {
+        margin-bottom: 0;
+    }
+
+    &-ProductName {
+        font-weight: 700;
+        font-size: 2rem;
+
+        @include mobile {
+            font-size: 1.8rem;
+        }
+    }
+
+    .Form {
+        max-width: 500px;
+
+        @include mobile {
+            max-width: none;
+        }
+
+        input,
+        textarea {
+            width: 100%;
+        }
+    }
+
+    &-ReviewRating {
+        margin-top: 1rem;
+    }
+
+    &-ReviewRatingCode {
+        font-size: 1.05rem;
+        font-weight: 700;
+        border: 0;
+        padding: 0;
+    }
+
+    &-RatingOptionGroup {
+        display: flex;
+
+        &::before {
+            content: '☆☆☆☆☆';
+            font-size: 3rem;
+            color: var(--color-primary-base);
+        }
+    }
+
+    .Field {
+        width: 100%;
+
+        &_type_radio {
+            margin-top: 0;
+            position: absolute;
+
+            label {
+                position: absolute;
+                display: block;
+                border: 0;
+                background-color: transparent;
```

Comment:
Let's no use `position: absolute` at all. Use text shadows, use `-webkit-text-stroke` or `text-shadow` to achieve outline if you really need it.

Reply:
Sorry maybe I'm missing something again but not sure how this would work as currently empty stars get overlaid by the filled ones this way. How would you like it to work instead?

Reply:
Should I once again do the complete implementation instead of you? :(